base/harmful/harmful.c

```diff
@@ -203,7 +203,8 @@ TRAP(lgammal)
 TRAP(nftw)
 TRAP(nl_langinfo)
 TRAP(putc_unlocked)
-TRAP(rand)
+//TRAP(rand) // Used in mongo-c-driver
```

Comment:
The driver has to be patched.

Reply:
The concerns have to be reported to MongoDB authors.

Reply:
Do you mean mongo-c-driver? Could you please refer to the issue, is it small patch?

Reply:
https://jira.mongodb.org/browse/CDRIVER-5646